pkg/assertions/yamlmeta.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package assertions
+
+import "github.com/vmware-tanzu/carvel-ytt/pkg/yamlmeta"
+
+// AddValidations appends validation Rules to node's validations metadata, later retrieved via GetValidations().
+func AddValidations(node yamlmeta.Node, rules []Rule) {
+	metas := node.GetMeta("validations")
```

Comment:
Magic string: let's extract to a constant for that added confidence that it's the same. 👍🏻